README.md

```diff
@@ -4,6 +4,21 @@ JustAssembly is a lightweight .NET assembly diff and analysis tool built on top
 
 ![alt text](https://d585tldpucybw.cloudfront.net/sfimages/default-source/productsimages/justassembly/how-it-works.png)
 
+## Command line usage
+
+It is possible to use JustAssembly with command line interface, in order to generate an XML file with assembly differences.
```

Comment:
To be more accurate - "assembly API differences".

Reply:
Yes...I totally agree with you

Reply:
Absolutely I agree with you